### PR TITLE
Also regenerate css files

### PR DESCRIPTION
**Pull request by amlmn**

### CODE REVIEW

File: regenerate.py [hunk 1]
```diff
@@ -27,10 +28,8 @@
 
 def run(*args) -> str:
   logging.debug('Running command: %s', args)
-  result = subprocess.run(args,
-                          stdout=subprocess.PIPE,
-                          stderr=subprocess.PIPE,
-                          encoding='utf-8')
+  result = subprocess.run(
```
Comment (amlmn) [inline regenerate.py]: @wbthomason I'm a little confused about why the automatic formatting changed. Which way does it format on your machine?

Comment (wbthomason) [reply regenerate.py]: The old way, i.e. stdout, etc. are all on their own line. I checked, and I'm running yapf with the repo .style.yapf

File: regenerate.py [hunk 2]
```diff
@@ -56,29 +54,44 @@ def output(input_path, output_path):
...
-                      help='output dir, relative to the repo root')
-  parser.add_argument('--pandoc_flags',
-                      type=str,
-                      default='',
-                      help='passed verbatim to pandoc')
-  parser.add_argument('--log_level',
-                      type=str,
-                      help='controls the script\'s logging level',
-                      default='ERROR')
+  parser.add_argument(
```
Comment (wbthomason) [inline regenerate.py]: My formatting for these changes is also like the old version.

**Review by wbthomason — COMMENTED** [review]
Looks good to me modulo the formatting weirdness. It'd be good but not critical to figure out why that's happening.

File: regenerate.py (hunk 1 above)
Comment (amlmn) [reply regenerate.py]: What version of yapf? (Not that it should really matter.)

I'm on 0.22.0, with a tabstop and shiftwidth of 2.

If you edit the source branch version of regenerate.py and then reformat, what happens?

It looks like the formatting changed in 5a5d53d. Do you remember what happened?

IIRC, you also have a local (non-repo) yapf config. Can you share the settings for it with me?

Comment (wbthomason) [reply regenerate.py]: `yapf 0.27.0`
Tabstop and shiftwidth are 2.
No change if I try to format the `source` version of this file.
I think 5a5d53d was when I fixed the problem where `yapf` was preferring my personal config, which is:

```
[style]
based_on_style = chromium
indent_width = 2
column_limit = 100
```

Comment (amlmn) [reply regenerate.py]: Okay, this should be fixed now. The 0.27.0 style is not backwards compatible with the 0.22.0 one. I upgraded my version of yapf to 0.27.0.

File: regenerate.py (hunk 2 above)
Comment (amlmn) [reply regenerate.py]: Fixed.

**Review by wbthomason — APPROVED** [review]
Looks good to me, thanks.

**Pull request merged**